src/atoms/Icon/Icon.native.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import icons from '../../icons';
+
+const iconSize = {
+  s: 10,
+  m: 16,
+  l: 20,
+};
+
+const Icon = ({ size, name, fill }) => {
+  const IconComponent = icons[name];
+  return <IconComponent fill={fill} height={iconSize[size]} width={iconSize[size]} />;
+};
+
+Icon.propTypes = {
+  size: PropTypes.oneOf(['s', 'm', 'l']),
```

Comment:
Shouldn't size be `small`, `medium` and `large` for Component props?

Reply:
We're following ```s, m, l``` pattern for font sizes. Wouldn't this be consistent with that?

Reply:
For internal usage, i.e tokens `s, m, l` is fine. I am talking about the API being exposed, a developer writing `<Icon size='small' />` seems more obvious and understandable than `<Icon size='s' />`. What do you say?

Reply:
Text is not only internal usage. We're using s,m,l there too. 

I agree that small, medium and large seems better. But then we have to keep it consistent across the DS.

Reply:
Yeah for `Text` also. Across all the react components of our DS.